linkerd/admin/src/main/scala/io/buoyant/linkerd/admin/DashboardHandler.scala

```diff
@@ -6,19 +6,22 @@ import com.twitter.util.Future
 import io.buoyant.linkerd.Build
 
 private[admin] class DashboardHandler extends Service[Request, Response] {
-  lazy val html = dashboardHtml
+  import DashboardHandler._
 
   override def apply(req: Request): Future[Response] = req.path match {
     case "/" =>
-      AdminHandler.mkResponse(html)
+      AdminHandler.mkResponse(dashboardHtml())
+    case rexp(router) =>
+      AdminHandler.mkResponse(dashboardHtml(router))
     case _ =>
       Future.value(Response(Status.NotFound))
   }
 
-  def dashboardHtml = {
+  def dashboardHtml(router: String = "") = {
     AdminHandler.html(
       content = s"""
         <div class="request-totals"></div>
+        <div class="router-name-data" data-router-name="${router}" style="visibility:hidden"></div>
```

Comment:
Hm, good question! I am not really sure what the most idiomatic way to pass these variables along is, between building up an object, and passing that, or using html5 data attributes. Since we're only passing 2 variables, it probably doesn't make too much of a difference. If someone has a strong opinion they can chime in :)

We could make global json object for our constants, something like `<script>var DASHBOARD_CONSTANTS = { router: "", linkerdVersion: ""}</script>`, and since our application's js is loaded after our `$content` it'd work... but if we ever moved those scripts, then the variables would be defined before they're used. If we did `<script id="data-router-name" type="application/json">$router</script>` we'd need to find the div to get the router name anyway, similar to what we're doing now.

Passing the data as part of the div does mean we're responsible for naming/obtaining the variables in the dashboard js code instead of here, which I kind of like, though it does mean we use an extra jquery selector to find the div.

We could make it its own js file, which we'd have to generate in the scala and then pass in in our list of `javaScripts` - but since we're only passing 2 variables it seems much. 

This did make me think, though, I don't need a separate div for the new data, so I'll just put all the data under one div, so I can reuse the jquery selector on the frontend.